Source/CarthageKit/Archive.swift

```diff
@@ -58,12 +58,12 @@ private func untargz(archive fileURL: URL, to destinationDirectoryURL: URL) -> S
 		.then(SignalProducer<(), CarthageError>.empty)
 }
 
-private let ArchiveTemplate = "carthage-archive.XXXXXX"
+private let archiveTemplate = "carthage-archive.XXXXXX"
```

Comment:
By "these" you mean "all" top-level constants? I need to turn off the rule "identifier_name" also. Sure about this? As far as I know it's Swift convention to keep even top-level constants small ...